demo/title/multisize/playground

```diff
@@ -0,0 +1,13 @@
+return (
+    <div>
+        <TitleMultisize preTitle="desde" title="250.000 €"  postTitle={['1.234 €/m', <sup>2</sup>]} titleSize="l"/>
```

Comment:
Yep, but the "<sup>" tag is not part of the multisize component itself, it is now written in the playground's definition (to emulate the final lookup of nc project UX design).
I know that when we create the component's wrapper we must add the key prop in the array's elements.
But for the moment, I will add it explicitly in the playground to remove this warning.